### ISSUE TITLE
nginx Detection and Handling with Docker

### ISSUE THREAD

**Issue by lbussy** [brewpi-remix/brewpi-tools-rmx#55]
When nginx is run in Docker, the script will still see it's installed, but it will fail to reconfigure it for an alternative port.  I need to allow installing Apache2 to an alternative port in these cases.

**Comment by thorrak** [comment]
You’ve decided to containerize BPR?

**Comment by lbussy** [comment]
No, this is the "John changed sh*t again so I need to change again" issue. :)

**Comment by thorrak** [comment]
You misspelled “improved”...
